cmd/keytransparency-sequencer/main.go

```diff
@@ -46,6 +51,11 @@ var (
 	logURL    = flag.String("log-url", "", "URL of Trillian Log Server for Signed Map Heads")
 	refresh   = flag.Duration("domain-refresh", 5*time.Second, "Time to detect new domain")
 	batchSize = flag.Int("batch-size", 100, "Maximum number of mutations to process per map revision")
+
+	etcdServers = flag.String("etcd_servers", "", "A comma-separated list of etcd servers; no etcd registration if empty")
+
+	lockDir     = flag.String("lock_file_path", "/test/multimaster", "etcd lock file directory path")
+	forceMaster = flag.Bool("force_master", false, "If true, assume master for all logs")
```

Comment:
Might want to document that it's not for production use?

Reply:
done